lib/SearchResultsList/SearchResultsList.js

```diff
@@ -208,6 +191,8 @@ const SearchResultsList = ({
       hidePageIndices={hidePageIndices}
       pagingCanGoNext={hasNextPage}
       pagingCanGoPrevious={hasPrevPage}
+      itemToView={itemToView}
+      onMarkPosition={onHandleRowFocus}
```

Comment:
`onHandleRowFocus` seems like we don't need both `on` and `handle` prefixes. Let's just call `onRowFocus` or `handleRowFocus`

Reply:
Thanks, fixed